src/main/resources/db/changelog/2022.10/06-01-changelog.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-latest.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet author="rlong (generated)" id="1665057691865-1">
+        <dropForeignKeyConstraint baseTableName="terr_question_mc" constraintName="FK_TERR_QUESTION_MC_ON_QUESTION_ID"/>
+    </changeSet>
+    <changeSet author="rlong (generated)" id="1665057691865-2">
```

Comment:
I like this, but with this in place is the change in deleteById still necessary? Seems like the questionMcRepository.deleteByQuestionId is no longer needed.

Reply:
Good catch! That was from when i was diagnosis the issue with foreign key constraint issues on deletes of MC questions. I will fix it.